import_logs.py

```diff
@@ -824,7 +825,10 @@ def _parse_args(self, option_parser):
         if not self.filenames:
             print(option_parser.format_help())
             sys.exit(1)
-
+				
+        if len(self.filenames) == 1 and '*' in self.filenames[0]:
```

Comment:
Why writing `len(self.filenames) == 1`, could we apply the glob when there are are more than 1 pattern specified?